### PR TITLE
remove HF token from build log

### PR DESCRIPTION
**Pull request by jiafuzha**

### CODE REVIEW

**Comment by jiafuzha** [comment]
@yutianchen666 
There is a ut test failure seems not related to my changes. Please help check below log.

https://github.com/intel/llm-on-ray/actions/runs/9543290317/job/26299745246?pr=253

**Comment by yutianchen666** [comment]
> @yutianchen666 There is a ut test failure seems not related to my changes. Please help check below log.
> 
> https://github.com/intel/llm-on-ray/actions/runs/9543290317/job/26299745246?pr=253

“NumPy 2.0 is planned to bereleased on June 16” https://numpy.org/news/. So the numpy-2.0.0 has a conflict with skimage

**Comment by jiafuzha** [comment]
CI has passed except the UT case tiancheng will submit a new PR to fix. please help review.

**Comment by yutianchen666** [comment]
I think in this PR it is possible to modify pyproject.toml [L23](https://github.com/intel/llm-on-ray/blob/cdce225cb2285baf9c151fecc7b6af853412e030/pyproject.toml#L23)directly of "numpy<2.0.0" to pass all CI. Because In order to adapt to latest numpy (2.0.0), I tried the versions of various installation packages, including Skimage, torch, ipex, oneccl_bind_pt, etc. However, there are a lot of incompatibilities, especially Python 3.9 does not support the latest Skimage, oneccl_bind_pt has not adapted to torch2.3, etc. So can only limit numpy to the < 2.0.0 version now

**Comment by jiafuzha** [comment]
> > @yutianchen666 There is a ut test failure seems not related to my changes. Please help check below log.
> > https://github.com/intel/llm-on-ray/actions/runs/9543290317/job/26299745246?pr=253
> 
> “NumPy 2.0 is planned to bereleased on June 16” https://numpy.org/news/. So the numpy-2.0.0 has a conflict with skimage

sounds good. let me try.

File: llm_on_ray/inference/models/vllm/llama-2-7b-chat-hf-vllm.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ ipex:
   enabled: false
   precision: bf16
 model_description:
-  model_id_or_path: meta-llama/Llama-2-7b-chat-hf
-  tokenizer_name_or_path: meta-llama/Llama-2-7b-chat-hf
+  model_id_or_path: NousResearch/Llama-2-7b-chat-hf
```
Comment (carsonwang) [inline llm_on_ray/inference/models/vllm/llama-2-7b-chat-hf-vllm.yaml:18]: can you revert all the changes for yamls in inference/models and still use meta-llama?

Comment (jiafuzha) [reply llm_on_ray/inference/models/vllm/llama-2-7b-chat-hf-vllm.yaml:18]: no, they are actually referenced by CI inference test cases.

**Review by carsonwang — APPROVED** [review]
Thanks for the quick fix.

**Pull request merged**